packages/vscode-extension/package.nls.json

```diff
@@ -154,8 +154,8 @@
     "teamstoolkit.commandsTreeViewProvider.manifestEditorTitleNew": "Edit manifest file",
     "teamstoolkit.commandsTreeViewProvider.previewACDescription": "Preview and author Adaptive Cards directly in Visual Studio Code",
     "teamstoolkit.commandsTreeViewProvider.previewAdaptiveCard": "Preview and Debug Adaptive Cards",
-    "teamstoolkit.commandsTreeViewProvider.previewDescription": "Preview your Teams app locally",
-    "teamstoolkit.commandsTreeViewProvider.previewTitle": "Preview your Teams app",
+    "teamstoolkit.commandsTreeViewProvider.previewDescription": "Debug and preview your Teams app",
+    "teamstoolkit.commandsTreeViewProvider.previewTitle": "Preview your Teams app (F5)",
```

Comment:
Better add a comment for the string to notify localization team that "F5" here is a shortcut key and no need to translate

Reply:
Thanks!